drivers/gps/nrf9160_gps/Kconfig

```diff
@@ -38,6 +38,24 @@ config NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 	  application send the AT commands to configure the GPS
 	  properly.
 
+config NRF9160_GPS_PRIORITY_WINDOW_TIMEOUT_SEC
+	int "Enable GPS priority window if the GPS is blocked for this duration"
+	default 10
+	help
+	  If the GPS is blocked for more than the amount of time specified by
+	  this option, and the GPS priority window is enabled in the passed in
+	  GPS configuration structure, the GPS priority window is requested by
```

Comment:
It might not be obvious to the reader what " the passed in GPS configuration strcuture" refers to, as it's tied to the `gps_start()` API documented elsewhere. I think it could be changed to something like 
`This configuration sets the amount of time the GPS must be blocked by LTE before requesting GPS priority. For GPS priority to be requested after this timeout, the GPS must also be configured with the priority flag enabled when starting it.`

Reply:
Otherwise the help text is now nice and helpful!

Reply:
Agreed, added your suggestion with some teaks.